test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,71 @@
+-- Requirement summary:
+-- [Data Resumption]:OnExitAllApplications(IGNITION_OFF) in terms of resumption 
+--
+-- Description:
+-- 1. HMI is running
+--    One App is registered and activated on HMI
+-- 2. Perform ignition Off
+--    HMI sends OnExitAllApplications(IGNITION_OFF)
+--
+-- Expected result:
+-- 1. SDL sends to App OnAppInterfaceUnregistered
+-- 2. SDL sends to HMI OnSDLClose and stops working
+
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
```

Comment:
@LitvinenkoIra , according to new design of commonFunctions and commonSteps, there is no different between them. So they should be merged into one.